app/services/api/v3/participant_declarations_query.rb

```diff
@@ -6,6 +6,8 @@ class ParticipantDeclarationsQuery
       include Api::Concerns::FilterCohorts
       include Api::Concerns::FilterUpdatedSince
 
+      RELEVANT_INDUCTION_STATUS = %w[active completed].freeze
```

Comment:
Maybe set this higher up or in a model maybe? so we don't need to declare `RELEVANT_INDUCTION_STATUS` twice in difference places?

Reply:
I'll set it to `Api::ParticipantDeclarations::Index::RELEVANT_INDUCTION_STATUS` that way its shared.

Reply:
I wonder if we should avoid that as those are services that support different versions of the API, and if we disable a version we would like to avoid extra dependencies? Maybe we can live with duplication or maybe moving it to the declaration model?

Reply:
I would prefer to keep the duplication instead of moving to declaration model, as its within the context.

Reply:
I agree, @leandroalemao what do you think?